src/main/scala/org/broadinstitute/dsde/firecloud/webservice/StatusApiService.scala

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.dsde.firecloud.webservice
+
+import java.text.SimpleDateFormat
+import java.util.Date
+
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.service._
+import spray.json._
+import spray.routing._
+
+trait StatusApiService extends HttpService with PerRequestCreator with FireCloudDirectives {
+
+  private final val ApiPrefix = "status"
```

Comment:
Dropped the `val` altogether - the Spray DSL lends itself to reading with inline literals, and that's what we do everywhere else